src/pages/study-room/hook/useStudyStepMutation.ts

```diff
@@ -0,0 +1,11 @@
+import { studyStep } from '@/apis/study-room/study-step';
+import { useMutation } from '@tanstack/react-query';
+
+const useStudyStepMutation = () => {
+  const studyStepMutation = useMutation({
+    mutationFn: studyStep,
+  });
+  return { studyStepMutation };
```

Comment:
요렇게 표현해줘도 괜찮지 않을까요?
지금 구조에서는 사용하는 쪽의 인터페이스가 일반적인 기대와 다를 것 같아요~
```suggestion
  return useMutation({
    mutationFn: studyStep,
  });
```